app/permissions/experiment.rb

```diff
@@ -97,7 +85,7 @@
   # protocol in module: read
   # step in module: read, read comments, read/download assets
   can :read_protocol_in_module do |user, protocol|
-    can_read_experiment?(user, protocol.my_module.experiment)
+    can_read_project_experiments?(user, protocol.my_module.experiment.project)
```

Comment:
I think it must be connected to can_read_my_module

Reply:
obsolete